data/random-teams.js

```diff
@@ -1902,11 +1902,11 @@ class RandomTeams extends Dex.ModdedDex {
 			gender: setData.set.gender || template.gender || (this.randomChance(1, 2) ? 'M' : 'F'),
 			item: items + '' || setData.set.item || '',
 			ability: abilities + '' || setData.set.ability || template.abilities['0'],
-			shiny: typeof setData.set.shiny === 'undefined' ? this.randomChance(1, 1024) : setData.set.shiny,
-			level: setData.set.level || 100,
+			shiny: setData.set.shiny || this.randomChance(1, 1024),
```

Comment:
Should be changed back in case there's some situation where `"shiny": false` is important based on event moves or something?